nmostesting/mocks/Node.py

```diff
@@ -66,6 +67,8 @@ def get_sender(self, stream_type="video"):
                 "active": True
             }
         }
+        global gl_media_type
+        gl_media_type = media_type
```

Comment:
Please don't do this. Change the `"manifest_href" to include the media type in the name (or even add it as a URL parameter, and access that in the route.